test/tests/functional/pbs_snapshot_unittest.py

```diff
@@ -58,11 +58,6 @@ class TestPBSSnapshot(TestFunctional):
     def setUp(self):
         TestFunctional.setUp(self)
 
-        # Create a custom resource called 'ngpus'
-        # This will help us test parts of PBSSnapUtils which handle resources
-        attr = {"type": "long", "flag": "nh"}
-        self.server.manager(MGR_CMD_CREATE, RSC, attr, id="ngpus", sudo=True)
```

Comment:
@dinesh1306 : the comment in your PR made it sound like you're replacing "ngpus" with "ngpus_snapshot" instead of just getting rid of it:

TestPBSSnapshot - all tests are failing on setup with error 'qmgr obj=ngpus svr=default: Duplicate entry in list ', 'qmgr: Error (15055) returned from server'. on cpuset machine custom resource 'ngpus' is already present due to while creating same name resource PTL got failed.
**Update resource name ngpus_snapshot in test suite setup.**

Reply:
Hi @bayucan, custom resource 'ngpus' not using anywhere in test suite. After offline discussion with @bhroam we decided to remove that resource.